book/src/integrations/vscode.md

```diff
@@ -1,10 +1,12 @@
-# VSCode extension
+# Visual Studio Code Code extension
 
-PRQL has a VSCode Extension that compiles a PRQL query in a VSCode editor and
-displays the resulting SQL code in a second pane on the side. This is very handy
-for editing, saving, and reusing PRQL queries in VScode.
+PRQL has a Visual Studio Code extension that compiles a PRQL query in a VS Code
```

Comment:
VS Code is more informal I think

Reply:
Yes, I think so too, but I figured that "Visual Studio Code" is the proper name so the first reference to it would be with its proper full name and any subsequent reference would be with the shorter more informal name "VS Code".

Reply:
OK good point